go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/golang/protobuf v1.4.2
 	github.com/pkg/errors v0.8.1
 	github.com/stretchr/testify v1.3.0
-	github.com/zondax/hid v0.9.0
+	github.com/troian/hid v0.9.3
```

Comment:
hi work!
would you mind sending a PR to zondax/hid for this instead?

Reply:
https://github.com/Zondax/hid/pull/3
appreciate if it could be merged asap